packages/realtime_client/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   http: '>=0.13.0 <2.0.0'
   logging: ^1.2.0
   meta: ^1.7.0
-  web_socket_channel: '>=2.3.0 <4.0.0'
+  web_socket_channel: '>=3.0.1 <4.0.0'
```

Comment:
This is a breaking change, so we can't do that, but if you use both if imports for `js_interop` and `html` this should still work, right?